lib/stdlib/src/dets.erl

```diff
@@ -3279,14 +3278,14 @@ fopen_existing_file(Tab, OpenArgs) ->
 		_Err ->
                     _ = file:close(Fd),
                     dets_utils:stop_disk_map(),
-                    error_logger:format("dets: compaction of file ~tp failed, "
-                                        "now repairing ...~n", [Fname]),
+                    ?LOG_ERROR("dets: compaction of file ~tp failed, "
+                                "now repairing ...~n", [Fname]),
```

Comment:
I perceive this as a recoverable error ? Should it then not be an info log?

Reply:
If the compaction of a dets table fails, that sounds like an error to me. The "repairing" part is more on the information side of things.

Reply:
I'm not sure if this is ok, there are other cases of repair for example here:
https://github.com/erlang/otp/blob/38437c143146eadddcf1720731a95558303a972e/lib/stdlib/src/dets.erl#L3254-L3264

This `{repair, _}` tuple then lands here:
https://github.com/erlang/otp/blob/38437c143146eadddcf1720731a95558303a972e/lib/stdlib/src/dets.erl#L3287-L3290

And is printed as `?LOG_INFO`, shouldn't we keep the compact case the same as others?

Reply:
Good point. Fixes.